### ISSUE TITLE
[Feature Request] Check-only mode for CI

### ISSUE THREAD

**Issue by PawelLipski** [thlorenz/doctoc#188]
Sorry if it's a duplicate... an option that would just **check if the ToCs are up-to-date** without modifying anything would be useful for the CI use.

Currently I need to do sth like that:

https://github.com/VirtusLab/git-machete-intellij-plugin/pull/636/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R14

**Comment by AndrewSouthpaw** [comment]
I think as a potential workaround, you could use the new `--update-only` flag that was introduced, and then see if there's any changes in the index?

**Comment by PawelLipski** [comment]
Yes, I'm currently doing sth like that... just need #189 to be addressed for that workaround to fully work :/

**Comment by kamadorueda** [comment]
If you want to run doctoc in the CI, ensuring tocs are up-to-date or fail the job if not, I highly suggest you this:

https://github.com/fluidattacks/makes#formatmarkdown

This is how a CI failure looks like:

https://github.com/fluidattacks/makes/pull/190#issuecomment-877709118

It can also be executed locally, which is nice

**Comment by AndrewSouthpaw** [comment]
Thanks @kamadorueda, that seems like a neat case for using nix. I'd say the vast majority of people don't use it, though, so I think there's still a use case for developing something like this.
